test/e2e/Makefile

```diff
@@ -7,21 +6,21 @@ docker:
 	docker build --tag tenderdash/e2e-node -f docker/Dockerfile ../..
 
 node:
-	go build -o build/node -tags badgerdb,boltdb,cleveldb,rocksdb ./node
+	go build -gcflags="all=-N -l" -o build/node -tags badgerdb,boltdb,cleveldb,rocksdb ./node
 
 e2e/app/compile:
 	docker run --rm -it --entrypoint "/src/tenderdash/test/e2e/entrypoint.sh" -w "/src/tenderdash/test/e2e" -v $(ROOT_PATH):/src/tenderdash tenderdash/e2e-node
 
-runner/simple: runner e2e/app/compile
+runner/simple: runner tests e2e/app/compile
```

Comment:
why are you running tests here?

Reply:
this network presents in upstream, we keep it here for consistency